src/System.Net.Http/tests/FunctionalTests/ManagedHandlerTest.cs

```diff
@@ -391,5 +392,103 @@ public sealed class ManagedHandler_HttpClientHandler_ConnectionPooling_Test : Ht
                 }
             }
         }
+
+        [Fact]
+        public async Task SmallConnectionTimeout_SubsequentRequestUsesDifferentConnection()
+        {
+            using (HttpClient client = CreateHttpClient())
+            using (var listener = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
```

Comment:
Any reason you didn't use the LoopbackServer test stuff here?

Reply:
No, I just copied the https://github.com/dotnet/corefx/pull/26839/files/0687e1a51ed8d57f0e2014068b45330f185e93e8#diff-4d205133c92dc4f27822fa53f3c12653R356 test and tweaked it.  Of course the question can then be asked recursively :)

Reply:
I'm fine either way.  I've been trying to use LoopbackServer just to minimize code.

Reply:
Yeah.  I'll go with it as it is in the PR, but then subsequently we can clean up all the tests in this file to use the loopback server.  The changes to all of them should be similar.

Reply:
Actually, it was easy enough to switch them over.  Doing as part of this PR.